sympy/sets/tests/test_sets.py

```diff
@@ -1035,21 +1035,14 @@ def test_issue_8257():
     assert FiniteSet(-oo) + Interval(-oo, oo) == reals_plus_negativeinfinity
 
 
-def test_issue_10285():
```

Comment:
refere [this comment](https://github.com/sympy/sympy/pull/11280#issuecomment-228707293)